nncf/__init__.py

```diff
@@ -46,7 +46,7 @@
 except ImportError:
     _LOADED_FRAMEWORKS["openvino"] = False
 
-from nncf.common.logging import nncf_logger
+from nncf.common.logging import nncf_logger  # pylint: disable=wrong-import-position
```

Comment:
?

Reply:
I'll move it to the top, I thought it some kind of special style for import applied here so `nncf_logger` is imported below